src/Client.cpp

```diff
@@ -996,3 +1025,40 @@ void CClient::ParseIdentifier(const CString& sAuthLine) {
 		m_sUser = sAuthLine;
 	}
 }
+
+void CClient::NotifyAvailableCaps()
+{
+	const CIRCSock *pSock = GetIRCSock();
+	if (!pSock)
+		return;
+
+	m_ssNotifiedCaps.clear();
+
+	const SCString& ssAvailCaps = pSock->GetAvailableCaps();
+	for (const CString& sCap : ssAvailCaps) {
```

Comment:
you can use `for (const CString& sCap : pSock->GetAvailableCaps())` directly... I don't think attaching it to a local variable changes much.